Odata-docs/odatacli/getting-started.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "OData Cli"
+description: "How to generate code using the odata-cli dotnet tool"
+
+author: ElizabethOkerio
+ms.author: ElizabethOkerio
+ms.date: 3/30/2022
+ms.topic: article
+ 
+---
+# OData CLI
+
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+**OData Cli** is a CLI tool that generates strongly-typed C# and Visual Basic client code for a specified OData service. It generates a `DataServiceContext` class to interact with the service and CLR types for each entity type and complex type in the service model.
+
+The extension is available at [Nuget](https://www.nuget.org/packages/Microsoft.OData.Cli/)
```

Comment:
Maybe it might be beneficial to show the full namespace to the user in case they ever want to google it:
```suggestion
The extension is available at [NuGet as Microsoft.OData.Cli](https://www.nuget.org/packages/Microsoft.OData.Cli/).
```